frontend/device/devicelistener.lua

```diff
@@ -313,6 +313,16 @@ function DeviceListener:onSwapPageTurnButtons()
     Device:invertButtons()
 end
 
+function DeviceListener:onSwapLeftPageTurnButtons()
```

Comment:
@hius07 any thoughts on adding an event?

Reply:
I don't think these new actions are worth to be added to Dispatcher, hence no need of proxy methods in DeviceListener.
All can be done in the menu elements module, with direct calls of `Device:`.

Reply:
I agree.

Reply:
is this what you mean?

```lua
table.insert(PhysicalButtons.sub_item_table, {
        text = _("Invert left side page-turn buttons"),
        checked_func = function()
            return G_reader_settings:isTrue("input_invert_left_page_turn_keys")
        end,
        callback = function()
            -- UIManager:broadcastEvent(Event:new("SwapLeftPageTurnButtons"))
            G_reader_settings:flipNilOrFalse("input_invert_left_page_turn_keys")
            Device:invertButtonsLeft()
        end,
    })
```